contracts/test/utils/Fixtures.sol

```diff
@@ -0,0 +1,59 @@
+import {Core} from "../../core/Core.sol";
```

Comment:
why does github not highlight this solidity?

Reply:
Sorted, missing `pragma` statement